themes/ddbasic/sass/components/cookie.scss

```diff
@@ -0,0 +1,79 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?